### PR TITLE
Feature/binary classifier fitting

### PR DESCRIPTION
**Pull request by rcehemann**
Fitting script binary_classifier_fitting.py is roughly finalized. I will just add more data and classifiers to improve the final model. The "runtime" script binary_classifier.py is set up so that we can make changes to the model (through the fitting script and pickle file) without having to adjust it. Loading and testing of amazon database is just a placeholder until we get the twitter DB set up.

### CODE REVIEW

File: py_feels/binary/binary_classifier.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# we only need pandas, mongo and joblib to use the model
+import pandas as pd
+from sklearn.externals.joblib import load
+
+# load fitted model
+model = load('./model/binary_classifier.pckl')
+
+# read amazon training data for testing loaded model
+amazon = pd.read_csv('./training/UCI_training_data/amazon_cells_labelled.txt',
```
Comment (thekeele) [inline py_feels/binary/binary_classifier.py]: What do you think about this style?
```
amazon = pd.read_csv(
    './training/UCI_training_data/amazon_cells_labelled.txt',
    sep="\t",
    names=['sentence', 'sentiment'],
    dtype={'sentence':str, 'sentiment':int}
)
```

File: py_feels/binary/binary_classifier.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+amazon = pd.read_csv('./training/UCI_training_data/amazon_cells_labelled.txt', 
+                     sep="\t", 
+                     names=['sentence', 'sentiment'], 
+                     dtype={'sentence':str, 'sentiment':int})
+
+# predict sentiments in the amazon data
+predictions = model.predict(amazon['sentence'])
+
+# print accuracy
+accuracy    = (predictions == amazon['sentiment']).sum()/predictions.size # 1 is perfect
```
Comment (thekeele) [inline py_feels/binary/binary_classifier.py]: `accuracy    =` extra whitespace

It's tempted to get things lined up but it's unsustainable.

File: py_feels/binary/training/binary_classifier_training.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                     sep="\t", 
+                     names=['sentence', 'sentiment'], 
+                     dtype={'sentence':str, 'sentiment':int})
+imdb   = pd.read_csv('UCI_training_data/imdb_labelled.txt', 
+                     sep="\t", 
+                     names=['sentence', 'sentiment'], 
+                     dtype={'sentence':str, 'sentiment':int})
+
+# concatenate the individual dataframes, shuffle them (sample with frac=1) and reset the index
+data   = pd.concat([amazon, yelp, imdb], ignore_index = True).sample(frac=1).reset_index(drop=True)
```
Comment (thekeele) [inline py_feels/binary/training/binary_classifier_training.py]: There are some really long lines in this file. This line should be the widest. 100 ruler width is a nice place to aim.

File: py_feels/binary/binary_classifier.py (hunk 1 above)
Comment (rcehemann) [reply py_feels/binary/binary_classifier.py]: looks nice, i'll work it in

File: py_feels/binary/binary_classifier.py (hunk 2 above)
Comment (rcehemann) [reply py_feels/binary/binary_classifier.py]: :(

File: py_feels/binary/training/binary_classifier_training.py (hunk 3 above)
Comment (rcehemann) [reply py_feels/binary/training/binary_classifier_training.py]: I'll split the long ones.  WRT the old directory, it's not on my local copy and i'm not sure why it appears on GH. I'll add the .DS_store files to .gitignore
